java/client/src/main/java/glide/api/commands/StreamBaseCommands.java

```diff
@@ -452,16 +452,15 @@ CompletableFuture<String> xgroupCreate(
      *     Map</code> is composed of a stream's key and the id of the entry after which the stream
      *     will be read. Use the special id of <code>{@literal ">"}</code> to receive only new messages.
      * @param group The consumer group name.
-     * @param consumer The newly created consumer.
+     * @param consumer The consumer name.
      * @return A <code>{@literal Map<String, Map<String, String[][]>>}</code> with stream
      *      keys, to <code>Map</code> of stream-ids, to an array of pairings with format <code>[[field, entry], [field, entry], ...]<code>.
-     *      Returns <code>null</code> if the consumer group does not exist. Returns a <code>Map</code> with a value of <code>null</code> if the stream is empty.
+     *      Returns <code>null</code> if the consumer group does not exist.
```

Comment:
If the consumer group doesn't exist, you will get a NOGROUP error. In the Python docs I added the following instead:
```suggestion
     *      Returns <code>null</code> if there is no stream that can be served.
```